README.md

```diff
@@ -1,8 +1,16 @@
-[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/255/11.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-currency-259)
+![Licence](https://img.shields.io/badge/licence-AGPL--3-blue.svg)
+[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/259/11.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-currency-259)
 [![Build Status](https://travis-ci.org/OCA/currency.svg?branch=11.0)](https://travis-ci.org/OCA/currency)
 [![codecov](https://codecov.io/gh/OCA/currency/branch/11.0/graph/badge.svg)](https://codecov.io/gh/OCA/currency)
 
 
 Currency
 ========
 Currency related Addons (crypto currencies, rate conversion, integrations with third party applications,...)
+
+List of Modules
```

Comment:
This is not put now on 11.0. It's not needed as branch are empty.

Reply:
@pedrobaeza I know but it will be populated automatic, no , when fetching translation or when the setup is added?

Reply:
As said, not needed as it doesn't add relevant information and makes to have a lot of extra commits.